hi-ml-cpath/src/health_cpath/utils/deepmil_utils.py

```diff
@@ -65,6 +65,18 @@ class EncoderParams(param.Parameterized):
     projection_dim: int = param.Integer(
         default=0, doc="If > 0, project the encoded tiles to this dimension. Otherwise, use identity projection."
     )
+    checkpoint_encoder: bool = param.Boolean(
+        default=False, doc="If True, checkpoint the encoder gradients during training. This is useful to trade compute "
+        "for memory, and is only supported for the Resnet18 and Resnet50 encoders at the moment. Default: False."
```

Comment:
same comment as above about the docs of `checkpoint_activations` can we keep the same name for the container parameter?